index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
My position on this matter may seem quirky, but I *think* it's justified.

  - I would like to remove Ramda as a dependency.<sup>†</sup>
  - I'm not opposed to dropping support for ES3.
  - I wouldn't support keeping the Ramda dependency *solely* to preserve ES3 compatibility.
  - We're not yet in a position to remove the Ramda dependency, so I see no reason to break ES3 compatibility here.

So, when we remove the Ramda dependency I'll be absolutely fine with making this change. Until then, though, I see no harm in taking advantage of the "insane" measures `R.trim` takes to support old browsers. :)

If we decide to add `S.trimLeft` and `S.trimRight` at some point, I'll reassess my position.

---

† _I even have a branch locally on which I attempted to remove Ramda. It's certainly possible, but a few things need to happen first. For example, sanctuary-def needs to expose its `show` function to save us from duplicating the definition of `R.toString` a second time._